### PR TITLE
Novemberbranch

### PR DESCRIPTION
**Pull request by AidaasinAyda**
Hi makeda, could you check the score file? And if possible upload the csvss

### CODE REVIEW

**Comment by drcandacemakedamoore** [comment]
The version of the script try_replace_colnames_scorecsvs.R pushed up does not accomplish it's mission. I suspect you pushed an early version. The script needs to go all the way through until you save a csv file or files which are the one you want.

**Review by AidaasinAyda — COMMENTED** [review]
@drcandacemakedamoore 
This is the new file. I wanted to create a pul request like last time, but the yellow status line with a green button didn't show up, so I'm trying to get your attention this way!

File: .gitignore
Comment (brickstudymasons) [inline .gitignore:1]: This change is fine.

File: brickstudy.Rproj
Comment (brickstudymasons) [inline brickstudy.Rproj:1]: I'm not sure you meant to push this.

**Comment by brickstudymasons** [comment]
You pushed changes on the .gitignore plus 6 new files here. I do not know if you meant me to review all 6 files. If not please open a PR with one single file, and perhaps the .gitignore changes you intend.

File: r_scripts/Script_regions_most_burden_WMH_Volbrain.R
Comment (brickstudymasons) [inline r_scripts/Script_regions_most_burden_WMH_Volbrain.R:1]: This is an important script, and should be in the repo.

File: r_scripts/fs_table_marjolein_abstract_28102024.R
Comment (brickstudymasons) [inline r_scripts/fs_table_marjolein_abstract_28102024.R:1]: Again, an important script that should be in the repo somewhere probably, but I'm not sure you meant to include it.

Comment (brickstudymasons) [reply r_scripts/fs_table_marjolein_abstract_28102024.R:1]: Also, you are trying to merge into main. This is incorrect @AidaasinAyda . Please make a pull request into develop with only the files you want for certain.

**Review by drcandacemakedamoore — CHANGES_REQUESTED** [review]
The pull request is onto main. Again...pull requests need to come off of develop, and go onto develop. I checked git log --branch, and this branch came from the main. Please make a PR from a branch off of develop, with only the files you want, then request to merge onto develop. Alternatively you can simply send me the files you want changed by email.

**Comment by drcandacemakedamoore** [comment]
replaced by properly branched

**Pull request closed without merging**